cross_list_courses/views.py

```diff
@@ -99,3 +105,46 @@ def create_new_pair(request):
 def delete_cross_listing(request, pk):
     remove_cross_listing(pk, request)
     return redirect('cross_list_courses:index')
+
+
+@lti_role_required(const.ADMINISTRATOR)
+@lti_permission_required(settings.PERMISSION_XLIST_TOOL)
+@require_http_methods(['GET'])
+def get_ci_data(request):
+    today = datetime.now()
+
+    if request.is_ajax():
+        q = request.GET.get('term', '')
+        logger.debug(q)
+        print q
+        courses = CourseInstance.objects.filter(Q(term__end_date__gte=today - timedelta(days=120))).filter(
+            Q(course_instance_id__icontains=q)|
+            Q(title__icontains=q)|
+            Q(sub_title__icontains=q) |
+            Q(short_title__icontains=q)).filter(Q(cs_class_type='E') |
+                                                Q(cs_class_type__isnull=True)).select_related('term')[:10]
```

Comment:
Should the slice be included?

Reply:
yes, 10 seems to be the number of entries we want to display in the dropdown. we don't want to display all. as they enter more characters, the results will filter down